cmd/loop/loopout.go

```diff
@@ -199,7 +199,14 @@ func loopOut(ctx *cli.Context) error {
 	if maxMinerFee == 0 {
 		maxMinerFee = limits.maxPrepayAmt
 	}
-	// Override the old maxMinerFee.
+
+	// Check that the maxMinerFee is sane. The prepay amount should also be
+	// greater then the minMinerFee.
+	if maxMinerFee < limits.minMinerFee {
```

Comment:
We could get into a state where our `prepay` is < `minMinerFee` (I get this in regtest), and no swaps will succeed. Not sure of how to address that 🤔

Reply:
Yep I got that too. Then this would mean the prepay is too low to cover the transaction fee, that prob should be handled in the prepay logic otherwise the server would be losing money.